field_audit/models.py

```diff
@@ -732,6 +735,11 @@ def delete(self, *, audit_action=AuditAction.RAISE):
         if audit_action is AuditAction.IGNORE:
             return super().delete()
         assert audit_action is AuditAction.AUDIT, audit_action
+
+        from .field_audit import is_audit_enabled
+        if not is_audit_enabled():
+            return super().delete()
```

Comment:
1. Why not do this check above?
```suggestion
        from .field_audit import is_audit_enabled
        if audit_action is AuditAction.IGNORE or not is_audit_enabled():
            return super().delete()
        assert audit_action is AuditAction.AUDIT, audit_action
```

2. Can the import be moved up to the top of the module? I see there are other local imports in this file, but wondering if this particular one is necessary?

These suggestions can be applied in all other changes in this file as well.